### PR TITLE
Deploy simplify prepare functions

### PR DESCRIPTION
**Pull request by rodrigosiqueira**
This PR is a follow-up from https://github.com/kworkflow/kworkflow/pull/441. Here, I'm trying to simplify the preparation functions in the deploy.

### CODE REVIEW

File: tests/deploy_test.sh [hunk 1]
```diff
@@ -734,38 +678,29 @@ function test_prepare_host_deploy_dir()
...
-    "ssh -p 2222 root@172.16.224.1 sudo \"mkdir -p /root/kw_deploy\""
-    "rsync -e 'ssh -p 2222' $KW_PLUGINS_DIR/kernel_install/debian.sh root@172.16.224.1:/root/kw_deploy/distro_deploy.sh -LrlptD --rsync-path='sudo rsync'"
-    "rsync -e 'ssh -p 2222' $KW_PLUGINS_DIR/kernel_install/remote_deploy.sh root@172.16.224.1:/root/kw_deploy/ -LrlptD --rsync-path='sudo rsync'"
-    "rsync -e 'ssh -p 2222' $KW_PLUGINS_DIR/kernel_install/utils.sh root@172.16.224.1:/root/kw_deploy/ -LrlptD --rsync-path='sudo rsync'"
-  )
+  expected_cmd="rsync -e 'ssh -p 2222' $scripts_path/$to_copy $target_address:/root/kw_deploy $rsync_flags --archive"
 
   output=$(prepare_remote_dir "$remote" "$port" "$user" "$flag")
-  compare_command_sequence 'expected_cmd_sequence' "$output" "$LINENO"
+  assert_equals_helper "Infer kernel image" "$LINENO" "$expected_cmd" "$output"
```
Comment (kwy95) [inline tests/deploy_test.sh:703]:
```suggestion
  assert_equals_helper 'Infer kernel image' "$LINENO" "$expected_cmd" "$output"
```

**Review by kwy95 — CHANGES_REQUESTED** [review]
Small nitpicks, otherwise looks good.

**Comment by rodrigosiqueira** [comment]
Thanks @kwy95 
PR updated.

**Review by kwy95 — APPROVED** [review]

**Comment by rodrigosiqueira** [comment]
Thanks @kwy95

**Pull request closed without merging**